### ISSUE TITLE
Take a screenshot of the SceneView with Widget projection

### ISSUE THREAD

**Issue by ivanjpg** [olexale/arkit_flutter_plugin#43]
I followed the example code for flutter's widget projection into a an `ARKitSceneView`. I can visualize correctly in screen. Then I tried to take a screenshot of what I saw in the screen, several questions in Swift/Objective-C points to the `snapshot()` function of the `SceneView` but I couldn't find it inside this plugin.

I also tried to use `Screenshot` plugin and write my own code using `RepaintBoundary` class and `toImage()` function. Both cases the screenshot is taken but only the projected flutter widget is seen and the background is entirely white. The camera scene is missing from the image.

Is there any way to do that?

**Comment by toonztudio** [comment]
Hi @ivanjpg,

I use https://pub.dev/packages/screenshot_share_image to take a screenshot and share. 
But this one cannot crop and reposition image, so I forked this one to a new plugin here https://pub.dev/packages/capture_and_share. In case you don't need to share but just save to image gallery you may want to fork it and use with some image gallery saver plugin.

**Comment by ivanjpg** [comment]
I tested both packages that you pointed me out. They work for me. I was testing over `arkit` (`SceneView`) and `camera` flutter's plugin, both works nice. The exception was `adv_camera` but no need to use it this time.

I also want to save the image to the photo gallery, so I took those plugins and made my own. In case someone's need: [native_screenshot](https://pub.dev/packages/native_screenshot).

Thank you so much @toonztudio.
